docs/DeployConfigurations.rst

```diff
@@ -55,7 +63,37 @@ The simplest configuration is when Nextcloud is installed on the host and Docker
 		class ExApp2 python
 		class ExApp3 python
 
-Suggested way to communicate with Docker: via ``socket``.
+Suggested way to communicate with Docker: via ``docker-socket-proxy``.
+
+.. mermaid::
+
+	stateDiagram-v2
+		classDef docker fill: #1f97ee, color: transparent, font-size: 34px, stroke: #364c53, stroke-width: 1px, background: url(https://raw.githubusercontent.com/cloud-py-api/app_api/main/docs/img/docker.png) no-repeat center center / contain
+		classDef nextcloud fill: #006aa3, color: transparent, font-size: 34px, stroke: #045987, stroke-width: 1px, background: url(https://raw.githubusercontent.com/cloud-py-api/app_api/main/docs/img/nextcloud.svg) no-repeat center center / contain
+		classDef python fill: #1e415f, color: white, stroke: #364c53, stroke-width: 1px
+
+		Host
+
+		state Host {
+			Nextcloud --> DockerSocketProxy: by port
+			Docker --> Containers
+			Docker --> DockerSocketProxy : /var/run/docker.sock
+
+			state Containers {
+				DockerSocketProxy --> ExApp1
+				DockerSocketProxy --> ExApp2
+				DockerSocketProxy --> ExApp3
+			}
+		}
+
+		class Nextcloud nextcloud
+		class Docker docker
+		class ExApp1 python
+		class ExApp2 python
+		class ExApp3 python
+
+
```

Comment:
cannot judge the changes above, (no preview available)

Reply:
screenshots would be cool!

Reply:
![Screenshot_4](https://github.com/cloud-py-api/app_api/assets/29692256/062fe2bc-11a3-43d5-bfce-e02d12b5feb0)